climatetree/frontendfinal
# Temp dev

### Change type:
- Bug fix / New feature

###Description:
- Fixes a bug by changing some code
- Implements a new screen
- Note to reviewers: <note goes here>

### Trello Card Link:
https://trello.com/c/xxxxxxxx

### PR Checklist:

### Steps to Verify This PR
- Login via Facebook/Google
- Goto Inspect -> Application -> localStorage on the left panel -> select our localhost -> check that Userid and JWT is present on logging
